learn-data-structures-by-building-a-shopping-cart/index.js

```diff
@@ -1,97 +1,207 @@
+const shoppingCartContainer = document.getElementById(
+  "shopping-cart-container"
+);
+const productsContainer = document.getElementById("products-container");
+const dessertCards = document.getElementById("dessert-card-container");
+const shoppingCartBtn = document.getElementById("cart-btn");
+const clearShoppingCartBtn = document.getElementById("clear-cart-btn");
+const totalNumberOfItems = document.getElementById("total-items");
+const cartSubTotal = document.getElementById("subtotal");
+const cartTaxes = document.getElementById("taxes");
+const cartTotal = document.getElementById("total");
+const showHideCartSpan = document.getElementById("show-hide-cart");
+let isCartShowing = false;
+
+/**
+ *
+ * We can decide if we want to provide this products array for them or write steps where the campers
+ * will build it out themselves
```

Comment:
I can see having learners create the array, then add two or three objects themselves. Then we could fill in the rest in the following step.